src/Mod/TechDraw/Gui/TaskDimension.ui

```diff
@@ -117,17 +111,10 @@ by negative value of 'Over Tolerance'.</string>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
-        <item row="0" column="0">
-         <widget class="QLabel" name="label_11">
-          <property name="text">
-           <string>Format Specifier:</string>
-          </property>
-         </widget>
-        </item>
-        <item row="0" column="1">
-         <widget class="QLineEdit" name="leFormatSpecifier">
+        <item row="3" column="1">
+         <widget class="QLineEdit" name="leFormatSpecifierUnderTolerance">
           <property name="toolTip">
-           <string>Text to be displayed</string>
+           <string>Specifies the undertolerance format in printf() style, or arbitrary text</string>
```

Comment:
We had a discussion about the tooltips a while ago and people complained the usage of developer-centric wording. I agreed since for example my colleagues don't know that "printf()" is but know how to create a technical drawing. Therefore I would omit printf and refer to the Wiki, for example:
"Specifies the undertolerance either formatted as described in the Wiki, or as arbitrary text"